kyverno.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: kyverno
+  version: 1.9.3
+  epoch: 0
+  description: Kubernetes Native Policy Management CLI
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kyverno/kyverno/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: dbc29ccb9a54479addc0f1c6c60879a50b9c20e0bd917cd6e0d123636f68e770
+
+  - uses: go/build
+    with:
+      packages: ./cmd/initContainer
+      ldflags: -X github.com/kyverno/kyverno/pkg/version.BuildVersion=${{package.version}}
+      output: kyvernopre
+
+  - uses: go/build
+    with:
+      packages: ./cmd/cleanup-controller
+      ldflags: -X github.com/kyverno/kyverno/pkg/version.BuildVersion=${{package.version}}
+      output: cleanup-controller
+
+  - uses: go/build
+    with:
+      packages: ./cmd/kyverno/
+      ldflags: -X github.com/kyverno/kyverno/pkg/version.BuildVersion=${{package.version}}
+      output: kyverno
```

Comment:
These could be subpackages, named for example `kyverno-init-container`, `kyverno-cleanup-controller`, and `kyverno-cli` as you're adding. The main package can remain `kyverno` which is built from `./cmd/kyverno`.